docs/userGuide/contentAuthoring.md

```diff
@@ -113,6 +113,24 @@ In addition, it supports:
 
 * More media blocks, embedding services and additional options can be found in [Markdown-it-block-embed docs](https://github.com/rotorz/markdown-it-block-embed)
 
+### Add your own variables
+
+In `_markbind/variables.md`, you can define your own variables which can be used anywhere in your site. The format is as follows.

```

Comment:
~`which`~`that`

https://en.oxforddictionaries.com/usage/that-or-which
Also note that "which" is usually preceded by a comma.